llvm/lib/Target/RISCV/RISCV.td

```diff
@@ -36,6 +36,7 @@ include "GISel/RISCVRegisterBanks.td"
 
 include "RISCVSchedRocket.td"
 include "RISCVSchedSiFive7.td"
```

Comment:
One question I should have asked before (not for this patch): Can we separate SiFive7 model into two files or two models (scalar scheduling and RVV scheduling)? I know the microarchitecture of X280 inherits from SiFive7, but there are some different designs I see from current scheduling model, which makes X280 and SiFive7 independent enough.

Maybe we can make scalar parts a multiclass and reuse it in X280 scheduling model.
cc @michaelmaitland

Reply:
> but there are some different designs I see from current scheduling model, which makes X280 and SiFive7 independent enough.

I'm not sure what different designs you're referring to here?

Reply:
OK, I may use the wrong words here. I meant that the CPU resources/pipeline are different, X280 has additional vector pipeline. Intuitively, they should be separated (I think). And, the outputs of `llvm-mca` contain vector resources even for scalar CPU like `sifive-u74`.

Reply:
I am currently exploring some refactors of the scheduler model in our downstream for better code reusability. I will take a look to see if what you suggest is possible. I see your point about the llvm-mca reporting. But besides llvm-mca, the suggested change doesn't impact scheduling. In my experience, the relevant TblGen classes may make this change harder than it is worth.